### DISCUSSION TITLE
When do usernames of deleted accounts become available again?

### DISCUSSION THREAD

**Discussion by torbendury** [community/community#53188] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello all! :slightly_smiling_face: 

I have a GitHub account (not this one) that I use at my employer. Since I had a very good inspiration, the account has a very funny name. But the company account only has the authorization (from my employer) to do administrative things. So I would like to create a corporate account with a different name to free up the funny one and use it privately.

So my question is, are usernames of deleted accounts freed again, if so, when?

**Comment by neutrinomuon** [comment]
Hi @torbendury, 

It looks like eventually the usernames of deleted accounts are freed up on GitHub. However, the exact time may vary... According to what I found in GitHub's doc:

"Recall that all repositories, forks of private repositories, wikis, issues, pull requests and GitHub Pages sites owned by your account will be deleted and your billing will end immediately, and your username will be available to anyone for use on GitHub after **90 days**."

You can find more information about account deletion and username availability in GitHub's help documentation here:

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account

I hope this helps you. Please, let me know.
